nncf/experimental/openvino_native/graph/transformations/command_creation.py

```diff
@@ -0,0 +1,32 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.graph.graph import NNCFNode
+from nncf.experimental.openvino_native.graph.transformations.commands import OVTargetPoint
+from nncf.experimental.openvino_native.graph.transformations.commands import OVBiasCorrectionCommand
+
+
+def create_bias_correction_command(node: NNCFNode, bias_value: np.ndarray) -> OVBiasCorrectionCommand:
+    """
+     Creates bias correction command.
+
+    :param node: The node in the NNCF graph that corresponds to operation with bias.
+    :param bias_value: The new bias value that will be set.
+    :return: The `OVBiasCorrectionCommand` command to update bias.
+    """
+    bias_port_id = node.layer_attributes.weight_port_id
+    target_point = OVTargetPoint(TargetType.LAYER, node.node_name, bias_port_id)
+    return OVBiasCorrectionCommand(target_point, bias_value)
```

Comment:
Is this code correct? I think it should be something like this
```python
add_node = nncf_graph.get_next_nodes(node_with_bias)[0]
bias_port_id = add_node.layer_attributes.weight_port_id
target_point = OVTargetPoint(TargetType.LAYER, node_with_bias.node_name, bias_port_id)
return OVBiasCorrectionCommand(target_point, bias_value)
```

Reply:
By the way, it seems like we can do this manipulation inside the `ModelTransformer` and preserve the method's signature (i.e. `create_bias_correction_command(node: NNCFNode, bias_value: np.ndarray)`).

Reply:
We can't save the signature, because the `bias_port_id` in the case of the `Convolution` node wouldn't be correct.